src/main/java/uk/gov/hmcts/reform/unspec/testingsupport/AssignCaseSupportController.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.unspec.testingsupport;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.unspec.service.CoreCaseUserService;
+
+@Api
+@Slf4j
+@RestController
+@AllArgsConstructor
+@RequestMapping(
+    path = "/testing-support",
+    produces = MediaType.APPLICATION_JSON_VALUE,
+    consumes = MediaType.APPLICATION_JSON_VALUE
+)
+public class AssignCaseSupportController {
```

Comment:
Can we add `@ConditionalOnExpression("${testing.support.enabled:false}")` here, so it won't be enabled by accident on prod? Do you think it would make sense to move the endpoint to `TestingSupportController` class?

Reply:
added conditional, it's in testing support package, separate class is making more sense